tests/acceptance/27_cf-secret/encrypt-no-key.cf

```diff
@@ -0,0 +1,69 @@
+##############################################################################
+#
+# Test that encrypting with no host- / key argument defaults to encrypting for
+# localhost.
+#
+##############################################################################
+
+body common control
+{
+    inputs => { "../default.cf.sub" };
+    bundlesequence => { default("$(this.promise_filename)") };
+    version => "1.0";
+}
+
+body delete init_delete
+{
+  dirlinks => "delete";
+  rmdirs   => "true";
+}
+
+bundle agent init
+{
+  methods:
+      "any"
+        usebundle => generate_key;
+      "any"
+        usebundle => trust_key;
+
+  files:
+    # Create plain text file
+      "$(G.testfile).plaintext"
+        create => "true",
+        content => "Hello World!";
+
+    # Delete encrypted file
+      "$(G.testfile).encrypted"
+        delete => init_delete;
```

Comment:
if the encrypted file is not present doesn't this error? I think you might want to add `if => fileexists( "$(G.testfile).encrypted)`